scripts/consensus_merge.py

```diff
@@ -520,14 +462,12 @@ def build_output_record(single_caller_variants, output_vcf, normal_cram, hotspot
     # For each caller, get information required for format fields for this variant
     for variant in single_caller_variants:
         normal_sample, tumor_sample = variant.record.samples
```

Comment:
Does this still have a use now that you're axing the `variant.normal` and `variant.tumor` sample classes?

Reply:
Also secondary thought: since you are passing in the normal cram you will be able to tell which sample is which because you can pull the sample info from the cram header.

Reply:
No, I just forgot to delete that line, good catch. 

Good to know about the sample name being in the cram header. For purposes of this script it no longer actually matters which is which in the tumor-normal sense, you just have to keep them straight by name. Formerly the means of getting the Strelka2 GT was different for tumor and normal, but now everything is the same.